Updated plugin.py for extract_ass

soultaco83 opened the pull request:
fixed extraction codec. Removed the subrip in the codec search that was left

soultaco83 committed:
Update plugin.py

fix extract. Removed the subrip in the codec search that was left

soultaco83 committed:
Update changelog.md

soultaco83 committed:
Update info.json

bump version

Josh5 commented:
This is your plugin right?
If so, can you update the info and docs to reflect that you are the maintainer and have links point to your repository for issues and support?

For example, the source for the SRT plugin is here:
https://github.com/Unmanic/plugin.extract_srt_subtitles_to_files

Thanks

soultaco83 committed:
Update changelog.md

soultaco83 committed:
Update info.json

soultaco83 committed:
Update plugin.py

soultaco83 committed:
Update plugin.py

soultaco83 commented:
@Josh5 You might be able to remove this if you want. I am working on creating a repo for a few plugins that I have modified myself. But I have also updated info.json as requested. And changed the Changelog to reflect this is version 1 of a fork from the extract SRT. Anything else I should change?

Josh5 commented:
@soultaco83 all good. Thanks. If you make your own repo and decide to just publish your plugins there, post a message in Discord, and we will help advertise your repo.
If you want me to remove this version down the line, just let me know also.
